web/server/vue-cli/src/mixins/to-csv.mixin.js

```diff
@@ -7,7 +7,13 @@ export default {
      * @param {string} fileName - default file name for export.
      */
     toCSV(data, fileName) {
-      const content = data.map(d => d.join(",") + "\r\n").join("");
+      const content = data
+        .map(d => d.join(",") + "\r\n")
+        .join("")
+        // Hashmark (#) is a valid URL character but it starts the hash
+        // fragment and for this reason it needs to be escaped.
+        .replace("#", encodeURIComponent("#"));
```

Comment:
The pattern in this function call is a "string" instead of a "regex", so the first occurrence will be replaced only.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace
Other note: encodeURIComponent("#") could be stored in a variable which provides that only one call will be done of this function.

Reply:
Yes, good catch. I change it to `replaceAll` so it will replace every occurances of hashmarks.

This is not frequently used function and encoding hashmark is not a slow operation, so I don't think it worth to move it into a separate variable.